src/PKSim.Presentation/DTO/Individuals/IndividualDefaultValueRetriever.cs

```diff
@@ -91,30 +92,48 @@ private void setDefaultValues(IndividualSettingsDTO individualSettingsDTO, Origi
          var parameterBMI = _individualModelTask.BMIBasedOn(originData, parameterWeight, parameterHeight);
 
          individualSettingsDTO.SetDefaultParameters(parameterAgeDTO,
-                                                    _parameterMapper.MapAsReadWriteFrom(parameterHeight),
-                                                    _parameterMapper.MapAsReadWriteFrom(parameterWeight),
-                                                    _parameterMapper.MapAsReadWriteFrom(parameterBMI),
-                                                    parameterGestationalAge);
+            _parameterMapper.MapAsReadWriteFrom(parameterHeight),
+            _parameterMapper.MapAsReadWriteFrom(parameterWeight),
+            _parameterMapper.MapAsReadWriteFrom(parameterBMI),
+            parameterGestationalAge);
       }
 
       public IndividualSettingsDTO DefaultSettings()
       {
-         return DefaultSettingForSpecies(DefaultSpecies());
+         return DefaultSettingFor(DefaultSpecies());
+      }
+
+      public IndividualSettingsDTO DefaultSettingFor(Species species)
+      {
+         var speciesPopulation = DefaultPopulationFor(species);
+         return defaultSettingsFor(species, speciesPopulation);
       }
 
-      public IndividualSettingsDTO DefaultSettingForSpecies(Species species)
+      public IndividualSettingsDTO DefaultSettingFor(SpeciesPopulation speciesPopulation)
```

Comment:
One test was failing because the deafult population was using gender male... but this does not exist for pregnancy obviously. So I updated the code to allow creating default settings for a given speciespopulation